### ISSUE TITLE
LF fichier levels.lvl

### ISSUE THREAD

**Issue by SathanBERNARD** [LoupLobet/px212#2]
Le fichier levels.lvl doit être en LF et non en CRLF (comme le donne le prof)

**Comment by SathanBERNARD** [comment]
sinon ça fait tout buguer

**Comment by LoupLobet** [comment]
SI @thonyan32 t'arrives à régler ton problème en bougeant sur linux ou avec WSL asap, ce serait nice.

**Comment by LoupLobet** [comment]
![image](https://user-images.githubusercontent.com/48242788/172051188-97e70a8d-6863-446a-ab88-06ae4889155b.png)

Car là quand j'ouvre un de tes fichiers avec mon éditeur, ça fait ça, et c'est assez embêtant.

**Comment by thonyan32** [comment]
@LoupLobet Yes , je suis passé à Kali , j'ai du du coup changer complètement le input mais c'est bon ca marche . Je vous envoie toutes les modifs dans la journée.

**Comment by LoupLobet** [comment]
ça marche merci

**Comment by LoupLobet** [comment]
@thonyan32 du coup c'est pas la peine du push sans les cr je l'ai fais 👍
